bindings_ffi/src/mls.rs

```diff
@@ -1185,18 +1185,29 @@ impl TryFrom<UserPreferenceUpdate> for FfiPreferenceUpdate {
     }
 }
 
-#[derive(uniffi::Object)]
+#[derive(uniffi::Object, Clone)]
 pub struct FfiConversation {
     inner: MlsGroup<RustXmtpClient>,
 }
 
 #[derive(uniffi::Object)]
-#[allow(unused_variables, dead_code)]
+#[allow(dead_code)]
```

Comment:
do we still need these `dead_code` attributes if we make the functions in the `impl` `pub`?

Reply:
Nice! Excited to get rid of that. 🔥